Client/MirObjects/ItemObject.cs

```diff
@@ -105,6 +119,29 @@ public override void DrawName()
             NameLabel.Draw();
         }
 
+        public void DrawGradeEff()
+        {
+            if (Grade > ItemGrade.Common)
```

Comment:
The grade doesn't change within the life of the ItemObject. This would be better being set once inside the Load(S.ObjectItem info) method - instead of being set to the same value every time it needs to update a frame.